README.md

```diff
@@ -1,17 +1,25 @@
-DustExtinction.jl
-=================
+# DustExtinction.jl
 
 [![Build Status](https://img.shields.io/travis/JuliaAstro/DustExtinction.jl.svg?style=flat-square)](https://travis-ci.org/JuliaAstro/DustExtinction.jl)
 
 
 Tools for interstellar dust extinction in astronomy
 
-Install
--------
+## Requirements
 
-```julia
-julia> Pkg.add("DustExtinction")
-```
+As of Feb 5, 2018, Julia 0.7+ is required.
+
+From a fresh Julia 0.7+ installation use the package manager (``` ]``` key) then do:
+
+``` add FITSIO```
```

Comment:
If a user doesn't already have these packages, won't they have to add them? I thought it would be helpful to include instructions on how to do that.

Reply:
No, the package manager takes care of it.  You don't have to manually add them, unless you want to explicit `using` them.

Reply:
This could be the result of a limit in my experience with Julia. In DustExtinction.jl there is the line "using FITSIO". Won't users need to have added OIFITS then?

Reply:
`FITSIO` is in the `REQUIRE` file, that's how the package manager knows that has to install `FITSIO` too :wink: